scripts/cob-shutdown

```diff
@@ -10,6 +10,9 @@ if [[ $HOSTNAME != *"b"* ]]; then
 	exit
 fi
 
+sudo /usr/sbin/cob-command stop -a
+sudo /usr/sbin/cob-command stop_core
```

Comment:
please remove all cob-commands from shutdown script. Ubuntu OS will handle this for us. It's enough to just call shutdown for all pcs